cloudformation/database/serverless-database.yaml

```diff
@@ -17,6 +17,9 @@ Metadata:
           - Vpc
           - DatabaseClusterSubnetOne
           - DatabaseClusterSubnetTwo
+          - VpcCidrBlock
+          - Subnet1CidrBlock
+          - Subnet2CidrBlock
```

Comment:
Would it make more sense to have the Cidr blocks for each resource placed right after the resource id field?

Reply:
I thought about it, and in my opinion having all CIDR block next to each other forces users to think about possible overlaps or inconsistencies in the blocks definition. What do you think?